pkgs/by-name/ve/vectorcode/package.nix

```diff
@@ -45,7 +46,51 @@ let
         dependencies = old.dependencies ++ [
           self.chroma-hnswlib
         ];
-        doCheck = false;
+
+        pytestFlags = [
+          # This package sometimes has issues running multiple tests in parallel.
+          # ValueError: An instance of Chroma already exists for ephemeral with different settings
+          "--numprocesses=1"
+        ];
+
+        # The base package disables additional tests, so explicitly override
+        disabledTests = [
+          # Tests are flaky / timing sensitive
+          "test_fastapi_server_token_authn_allows_when_it_should_allow"
+          "test_fastapi_server_token_authn_rejects_when_it_should_reject"
+
+          # Issue with event loop
+          "test_http_client_bw_compatibility"
+
+          # httpx ReadError
+          "test_not_existing_collection_delete"
+        ];
+
+        disabledTestPaths = [
+          # Tests require network access
+          "chromadb/test/auth/test_simple_rbac_authz.py"
+          "chromadb/test/db/test_system.py"
+          "chromadb/test/ef/test_default_ef.py"
+          "chromadb/test/property/"
+          "chromadb/test/property/test_cross_version_persist.py"
+          "chromadb/test/stress/"
+          "chromadb/test/test_api.py"
+
+          # httpx failures
+          "chromadb/test/api/test_delete_database.py"
+
+          # Cannot be loaded by pytest without path hacks (fixed in 1.0.0)
+          "chromadb/test/test_logservice.py"
+          "chromadb/test/proto/test_utils.py"
+          "chromadb/test/segment/distributed/test_protobuf_translation.py"
+
+          # Hypothesis FailedHealthCheck due to nested @given tests
+          "chromadb/test/cache/test_cache.py"
+        ];
+
+        meta = old.meta // {
+          broken = stdenv.hostPlatform.isLinux && stdenv.hostPlatform.isAarch64;
```

Comment:
Same question: Should I move this to [pkgs/development/python-modules/chromadb/default.nix](https://github.com/NixOS/nixpkgs/blob/9541941f0405c4d1145923a436b205458d0b4d2a/pkgs/development/python-modules/chromadb/default.nix) ?